contracts/BalanceTrackerBase.sol

```diff
@@ -246,13 +246,11 @@ abstract contract BalanceTrackerBase {
 
         // Get total mech and requester delivery rates
         uint256 totalMechDeliveryRate;
-        uint256 totalRequesterDeliveryRate;
         uint256 totalRateDiff;
         for (uint256 i = 0; i < deliveredRequests.length; ++i) {
             // Check if request was delivered
             if (deliveredRequests[i]) {
                 totalMechDeliveryRate += mechDeliveryRates[i];
-                totalRequesterDeliveryRate += requesterDeliveryRates[i];
```

Comment:
Not actually needed, it reflects delivery rates of all the requesters in this batch, not relevant.